### PR TITLE
basic scrapper setup and declaration of item fields to scrap for

### PR DESCRIPTION
**Pull request by YangLeoZhao**
basic setup. Testing merging. 
Take a look at scrapper/scrapper/items.py and make sure all the fields are inside.

Please approve @dennisso @bleskowsky

### CODE REVIEW

File: scrapper/scrapper/items.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    fit_neck = scrapy.Field()
+    fit_chest = scrapy.Field()
+    fit_arm = scrapy.Field()
+
+    #Not sure if needed
+    gender = scrapy.Field()
+    cotton = scrapy.Field()
+
+    #Boolean fields indicating if the shirt belongs in the category
+    t_shirt = scrapy.Field()
```
Comment (sodennis) [inline scrapper/scrapper/items.py]: use enum for this.

File: scrapper/scrapper/items.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+class ScrapperItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
+
+class menTop(scrapy.Item):
+    #Attributes to scrap
+    name = scrapy.Field()
+    price = scrapy.Field()
+    fit_neck = scrapy.Field()
```
Comment (bleskowsky) [inline scrapper/scrapper/items.py]: How would we scrape the fit_\* variables?

**Comment by bleskowsky** [comment]
Maybe add description and brand fields?

File: scrapper/scrapper/items.py (hunk 1 above)
Comment (YangLeoZhao) [inline scrapper/scrapper/items.py]: do you mean t_shirt = 1, graphic_tee = 2...

File: scrapper/scrapper/items.py (hunk 2 above)
Comment (YangLeoZhao) [inline scrapper/scrapper/items.py]: So fit_neck would contain a dict like {'small':[10,20], "medium":[20,30]}
These values are on the american eagle website. You can see it if you scroll down on this link: https://www.ae.com/web/browse/product_details.jsp?productId=0153_9141_400&catId=cat6300043

**Comment by YangLeoZhao** [comment]
description, product url, pic url, brand should all be added

File: scrapper/scrapper/items.py (hunk 1 above)
Comment (sodennis) [inline scrapper/scrapper/items.py]: yeah so tshirt type will be shirt_type = ShirtType.t_shirt

Comment (YangLeoZhao) [inline scrapper/scrapper/items.py]: Actually.. there is no need to store all the bools here. I should just store a shirt_type which we extract from the scrapper.

Comment (sodennis) [inline scrapper/scrapper/items.py]: yup change that and you're good to push

**Pull request merged**